frontend/src/components/FilterPanel/FilterPanel.jsx

```diff
@@ -91,7 +91,15 @@ export default function FilterPanel() {
             value={kilometer}
             onChange={hkilometerChange}
           />
-          <label htmlFor="volume">Kilomètres {kilometer}</label>
+          <input
+            type="text"
+            className="kmInput"
+            value={kilometer}
+            onChange={hkilometerChange}
+          />
+          <label className="kilometer" htmlFor="volume">
```

Comment:
l'attribut htmlFor va fonctionner mais on utilise plutôt juste 'for' maintenant ! :)